### PR TITLE
Setup Section based Native ads (all sites)

### PR DESCRIPTION
**Pull request by jwade1327**
Screen Shot 2024-01-05 at 2 11 15 PM
Screen Shot 2024-01-05 at 2 11 08 PM

### CODE REVIEW

**Comment by jwade1327** [comment]
[Setup Section based NativeX ads](https://trello.com/c/joYyBHuA/471-setup-section-based-nativex-ads)

File: packages/global/components/layouts/content/default.marko [hunk 1]
```diff
@@ -233,7 +233,7 @@ $ const shouldInjectAds = displayAds && ["article", "video", "news", "podcast"].
               excludeContentTypes,
             }
           >
-          <@native-x indexes=[0] name="default" aliases=aliases />
+            <@native-x indexes=[0] name="related-content" aliases=aliases />
```
Comment (Shinsina) [inline packages/global/components/layouts/content/default.marko:236]: This appears to be in the "Latest In..." block is this intended to use the same Native-X placement as related content?

**Review by Shinsina — CHANGES_REQUESTED** [review]
See other comment, unsure we subsequently need to add default positions for all those newly added aliases or not.

File: packages/global/components/layouts/content/default.marko (hunk 1 above)
Comment (jwade1327) [reply packages/global/components/layouts/content/default.marko:236]: Yes. It's going to be in the right rail as well as the bottom related block (same as we have for SMG)

**Review by Shinsina — APPROVED** [review]

**Pull request merged**